sos/report/plugins/storcli.py

```diff
@@ -36,9 +40,14 @@ def setup(self):
 
         logpath = self.get_cmd_output_path()
 
+        json = ''
+        if self.get_option('json'):
+            json = ' J'
```

Comment:
Maybe just subjective stylistic option: can't it be shortened to

    json = ' J' if self.get_option('json') else ''

?

Reply:
I actually think the one-liner is more readable, personally.

+1 for shortening, then ack from me.

Reply:
@TurboTurtle ,
Incorporated the change. PTAL.